src/content/docs/en/basics/astro-components.mdx

```diff
@@ -291,7 +291,7 @@ Named slots can also be passed to [UI framework components](/en/guides/framework
 
 
 :::note
-Dynamically generating an Astro slot name, such as within a map function, was added in Astro v4.2.0. For earlier versions of Astro, it might be best to generate these dynamic slots within the framework itself.
+An astro slot name can not be dynamically generated, such as within a map function. If this feature is needed within UI framework components, it might be best to generate these dynamic slots within the framework itself.
```

Comment:
```suggestion
It is not possible to dynamically generate an Astro slot name, such as within a map function. If this feature is needed within UI framework components, it might be best to generate these dynamic slots within the framework itself.
```
I know you just faithfully reproduced the old message, but I'd improve a little now that I have the chance! (since search hits tend to be for e.g. "How do I dynamically generate an Astro slot name"